src/shared/logger/activation.ts

```diff
@@ -21,6 +21,8 @@ const localize = nls.loadMessageBundle()
 
 const DEFAULT_LOG_LEVEL: LogLevel = 'info'
 
+const ONE_HUNDRED_MEGABYTES = 100000000
```

Comment:
- Do not spell out common, unambiguous abbreviations such as "megabytes". 
    - "bytes" abbreviation is not typical so it should be expanded.
- Variable names should usually describe the semantics. 

```suggestion
const LOG_MAX_BYTES = 100000000  // 100 MB
```

Reply:
Thanks